vm/host/teller.go

```diff
@@ -162,6 +162,7 @@ func (t *Teller) DoPay(witness string, gasRatio int64) (paidGas *common.Fixed, e
 				return
 			}
 			t.h.db.SetTokenBalance("ram", ramPayer, currentRAM-ram)
+			t.h.db.ChangeUsedRAMInfo(ramPayer, ram)
```

Comment:
delta should be -ram?

Reply:
It is `used` ram. More means more ram has been used